src/transformers/modeling_utils.py

```diff
@@ -2497,6 +2511,101 @@ def _check_and_enable_flash_attn_2(
             config._attn_implementation = "flash_attention_2"
         return config
 
+    @classmethod
+    def _check_and_enable_flash_attn_3(
```

Comment:
this needs an update to remove all of the stuff related to better transforemrs, and only keep stuff specific to FA3